app-typescript/components/TreeSelect/TreeSelectItem.tsx

```diff
@@ -25,7 +26,10 @@ export class TreeSelectItem<T> extends React.Component<IProps<T>> {
     render() {
         return (
             <li
-                className='suggestion-item suggestion-item--multi-select'
+                className={
+                    'suggestion-item suggestion-item--multi-select'
+                    + (this.props.labelWrap ? ' suggestion-item--wrap' : '')
```

Comment:
how about making it wrap unconditionally?

Reply:
but why?